doc/contract-update.md

```diff
@@ -0,0 +1,68 @@
+# Upgrading and Migrating contracts
```

Comment:
Can we remove this please?
Will be confusing because we are implementing the delegateCall approach now

Reply:
Can be handled in PR2